examples/client/Locomotion/src/I18n/index.js

```diff
@@ -1,22 +1,146 @@
 import i18n from 'i18next';
+import _ from 'lodash';
+import * as axios from 'axios';
 import { initReactI18next } from 'react-i18next';
+import i18nHttpLoader from 'i18next-http-backend';
+import Backend from '@autofleet/i18next-remote-backend-with-locals';
+import * as RNLocalize from 'react-native-localize';
+import moment from 'moment';
+import 'moment/locale/es';
+import 'moment/locale/el';
+import Config from 'react-native-config';
+import Mixpanel from '../services/Mixpanel';
+import { StorageService } from '../services';
 
 import en from './en.json';
+import el from './el.json';
+// import es from './es.json';
+
+const USER_LANGUAGE_STORAGE_KEY = 'userLanguage';
+
+const DEFAULT_LANGUAGE_CODE = 'en';
+
+const getUserLanguage = async () => {
+  const languageCode = await StorageService.get(USER_LANGUAGE_STORAGE_KEY);
+  return languageCode;
+};
+
+const extractLanguageFromUrl = (url) => {
+  if (!url) {
+    return DEFAULT_LANGUAGE_CODE;
+  }
+  const endpoints = url.split('/');
+  const lastEndpoint = endpoints[endpoints.length - 1];
+  return lastEndpoint.split('.')[0];
+};
+
+const updateUserLanguage = chosenLanguageCode => StorageService.save(
+  { [USER_LANGUAGE_STORAGE_KEY]: chosenLanguageCode },
+);
+
+let userLanguage;
+
+export const getUserLanguageCode = () => userLanguage;
+
+export const supportedLanguages = {
+  en: {
+    label: 'English',
+    translation: en,
+  },
+  el: {
+    label: 'Ελληνικά',
+    translation: el,
+  },
+  // es: {
+  //   label: 'Español',
+  //   translation: es,
+  // },
+};
+
+export const getPreferredLanguageCode = async () => (
+  (await getUserLanguage())
+  || (Object.keys(supportedLanguages).includes(RNLocalize.getLocales()[0].languageCode)
+    ? RNLocalize.getLocales()[0].languageCode
+    : DEFAULT_LANGUAGE_CODE)
+);
+
+(async () => {
+  userLanguage = await getPreferredLanguageCode();
+})();
+
+
+const languageDetector = {
+  type: 'languageDetector',
+  async: true, // async detection
+  // eslint-disable-next-line no-return-await
+  detect: async cb => cb(await getPreferredLanguageCode()),
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  init: () => {},
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  cacheUserLanguage: () => {},
```

Comment:
do we have to define it? maybe we can to delete instead of disable linter

Reply:
crashing w/o it